diff --git a/node_modules/vitepress/dist/client/app/index.js b/node_modules/vitepress/dist/client/app/index.js
index 5463ae9..54cdff5 100644
--- a/node_modules/vitepress/dist/client/app/index.js
+++ b/node_modules/vitepress/dist/client/app/index.js
@@ -97,6 +97,15 @@ function newRouter() {
     return createRouter((path) => {
         let pageFilePath = pathToFile(path);
         let pageModule = null;
+
+        if (inBrowser && !pageFilePath) {
+            const locale = path.split('/')[1]
+            if (__locales__.includes(locale)) {
+                isInitialPageLoad = false
+                pageFilePath = pathToFile(path.slice(locale.length + 1)) + '?fallback=' + locale
+            }
+        }
+
         if (pageFilePath) {
             // use lean build if this is the initial page load
             if (isInitialPageLoad) {
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPNavBar.vue b/node_modules/vitepress/dist/client/theme-default/components/VPNavBar.vue
index d1fbe36..1edd6db 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPNavBar.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPNavBar.vue
@@ -81,7 +81,6 @@ watchPostEffect(() => {
 .VPNavBar.screen-open {
   transition: none;
   background-color: var(--vp-nav-bg-color);
-  border-bottom: 1px solid var(--vp-c-divider);
 }
 
 .VPNavBar:not(.home) {
@@ -172,16 +171,22 @@ watchPostEffect(() => {
   .VPNavBar.has-sidebar .content {
     position: relative;
     z-index: 1;
-    padding-right: 32px;
     padding-left: var(--vp-sidebar-width);
   }
+
+  .VPNavBar.has-sidebar .content-body {
+    padding-right: 32px;
+  }
 }
 
 @media (min-width: 1440px) {
   .VPNavBar.has-sidebar .content {
-    padding-right: calc((100vw - var(--vp-layout-max-width)) / 2 + 32px);
     padding-left: calc((100vw - var(--vp-layout-max-width)) / 2 + var(--vp-sidebar-width));
   }
+
+  .VPNavBar.has-sidebar .content-body {
+    padding-right: calc((100vw - var(--vp-layout-max-width)) / 2 + 32px);
+  }
 }
 
 .content-body {
diff --git a/node_modules/vitepress/dist/client/theme-default/components/VPSidebar.vue b/node_modules/vitepress/dist/client/theme-default/components/VPSidebar.vue
index 35fa783..10ea85b 100644
--- a/node_modules/vitepress/dist/client/theme-default/components/VPSidebar.vue
+++ b/node_modules/vitepress/dist/client/theme-default/components/VPSidebar.vue
@@ -111,8 +111,8 @@ watch(
 
 @media (min-width: 1440px) {
   .VPSidebar {
-    padding-left: max(32px, calc((100% - (var(--vp-layout-max-width) - 64px)) / 2));
-    width: calc((100% - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
+    padding-left: max(32px, calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2));
+    width: calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
   }
 }
 
diff --git a/node_modules/vitepress/dist/node/chunk-BBDUp32J.js b/node_modules/vitepress/dist/node/chunk-BBDUp32J.js
index 3b1fb75..8e9466e 100644
--- a/node_modules/vitepress/dist/node/chunk-BBDUp32J.js
+++ b/node_modules/vitepress/dist/node/chunk-BBDUp32J.js
@@ -37172,7 +37172,11 @@ function injectPageDataCode(tags, data) {
   const code = `
 export const __pageData = JSON.parse(${JSON.stringify(
     JSON.stringify(data)
-  )})`;
+  )})
+  if (import.meta.url.includes('?fallback=')) {
+    __pageData.relativePath = import.meta.url.slice(import.meta.url.lastIndexOf('=') + 1) + '/' + __pageData.relativePath
+  }
+  `;
   const existingScriptIndex = tags.findIndex((tag) => {
     return scriptRE.test(tag) && !scriptSetupRE.test(tag) && !scriptClientRE$1.test(tag);
   });
